.github/workflows/ci.yml

```
@@ -96,149 +80,176 @@ jobs:
+          checked_out="$(git rev-parse HEAD)"
+          if [ "$checked_out" != "$GITHUB_SHA" ]; then
+            echo "release checkout mismatch: expected $GITHUB_SHA, got $checked_out" >&2
+            exit 1
+          fi
+
       - uses: actions/setup-node@v4
         with:
           node-version: "24"
-          registry-url: "https://registry.npmjs.org"
+          registry-url: https://registry.npmjs.org
           package-manager-cache: false
```

actionlint (1.7.12)

[error] input "package-manager-cache" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)